assets/src/edit-story/app/media/media3p/categories/typedefs.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Categories actions typedef
+ *
+ * @typedef {(payload: {provider: string}) => undefined} DeselectCategory
+ * @typedef {(payload: {provider: string}) => undefined} FetchCategoriesStart
+ * @typedef {(payload: {provider: string}) => undefined} FetchCategoriesError
+ * @typedef {(payload: {provider: string, categories: *}) => undefined} FetchCategoriesSuccess
```

Comment:
Is it possible to type 'categories' as list of Category?

Reply:
Done